src/core/model_config.proto

```diff
@@ -155,15 +155,15 @@ message ModelInstanceGroup
   //@@
   repeated int32 gpus = 3;
 
-  //@@  .. cpp:var:: string profile
+  //@@  .. cpp:var:: string profile (repeated)
   //@@
   //@@     For TensorRT models, using inputs with dynamic shape, this
```

Comment:
Description needs update. I think it needs to say the set of optimization profiles available to each instance and server will choose from those the optimal profile based on the shape of the input tensors.